<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>559</width>
      <height>297</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="sensor_three_label">
       <property name="text">
        <string>Sensor Three</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLineEdit" name="sensor_two_name">
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::VisualMoveStyle</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="sensor_two_label">
       <property name="text">
        <string>Sensor Two</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QCheckBox" name="check_four">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QSlider" name="sample_rate">
       <property name="whatsThis">
        <string>p</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="logging_button">
       <property name="text">
        <string>Start Logging</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="sensor_one_name">
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::VisualMoveStyle</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="sensor_three_name">
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::VisualMoveStyle</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QCheckBox" name="check_two">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPushButton" name="save_setting_button">
       <property name="text">
        <string>Save Settings</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLabel" name="sample_rate_label">
       <property name="text">
        <string>Sample Rate</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="sensor_four_label">
       <property name="text">
        <string>Sensor Four</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QCheckBox" name="check_three">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="sensor_four_name">
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::VisualMoveStyle</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="sensor_one_label">
       <property name="text">
        <string>Sensor One</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="check_one">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="initialize_card_button">
       <property name="text">
        <string>Initialize Card</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QComboBox" name="filetype">
       <item>
        <property name="text">
         <string>CSV</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Raw</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="schedule_button">
       <property name="text">
        <string>Start Scheduling</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QDateTimeEdit" name="start_time">
       <property name="displayFormat">
        <string>M/d/yy h:mm:ss AP</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLCDNumber" name="sample_rate_display">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="smallDecimalPoint">
        <bool>true</bool>
       </property>
       <property name="digitCount">
        <number>5</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="6" column="3" colspan="2">
      <widget class="QPushButton" name="file_save_button">
       <property name="text">
        <string>File Save</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3" colspan="2">
      <widget class="QPushButton" name="file_load_button">
       <property name="text">
        <string>File Load</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3" colspan="2">
      <widget class="QDateTimeEdit" name="end_time">
       <property name="displayFormat">
        <string>M/d/yy h:mm:ss AP</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>sensor_one_name</tabstop>
  <tabstop>check_one</tabstop>
  <tabstop>sensor_two_name</tabstop>
  <tabstop>check_two</tabstop>
  <tabstop>sensor_three_name</tabstop>
  <tabstop>check_three</tabstop>
  <tabstop>sensor_four_name</tabstop>
  <tabstop>check_four</tabstop>
  <tabstop>sample_rate</tabstop>
  <tabstop>initialize_card_button</tabstop>
  <tabstop>logging_button</tabstop>
  <tabstop>file_save_button</tabstop>
  <tabstop>save_setting_button</tabstop>
  <tabstop>filetype</tabstop>
  <tabstop>file_load_button</tabstop>
  <tabstop>start_time</tabstop>
  <tabstop>schedule_button</tabstop>
  <tabstop>end_time</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>sample_rate</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sample_rate_display</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
